### PR TITLE
restrict TLS to 1.2 or 1.l

### PR DESCRIPTION
**Pull request by voidlock**
TLS 1.0 is considered an early TLS, and is over 15 years old. PCI will
prohibit the use of 1.0 after June 2016.

This just sets some good patterns in place early on

### CODE REVIEW

File: src/logplex.app.src [hunk 1]
```diff
@@ -81,6 +81,7 @@
     ,{http_drain_max_ttl, 18000000} % ms
     ,{workers, 10}
     ,{tls_mode, secure} % OpenSSL defaults to 100
+    ,{tls_versions, ['tlsv1.2', 'tlsv1.1']}  % Successfully TLS versions, see ssl:versions()
```
Comment (srid) [inline src/logplex.app.src]: Typo at `Successfully`

**Comment by srid** [comment]
Should not `approved_versions` be invoked from somewhere?

**Comment by voidlock** [comment]
It is called in the `connect_opts/3` function

**Comment by srid** [comment]
Yup, missed it in the diff.

:shipit:

**Pull request merged**